app/controllers/dogs_controller.rb

```diff
@@ -60,10 +60,15 @@ class DogsController < ApplicationController
   before_action :edit_dog_check, only: %i(edit update)
 
   def index
-    is_manager = signed_in? && session[:mgr_view]
+    is_manager = signed_in?
```

Comment:
hmm, should be okay.  However the title will probably always display as 'Dog Manager' when someone is signed in.  Probably overloading the use of ```is_manager``` 

Maybe something like

 ```@title = session[:mgr_view] ? 'Dog Manager' : 'Our Dogs' ```

In addition to your change would do it?